model-optimizer/extensions/front/onnx/pad_ext.py

```diff
@@ -28,11 +29,13 @@ class PadFrontExtractor(FrontExtractorOp):
     @classmethod
     def extract(cls, node):
         mode = onnx_attr(node, 'mode', 's', default='constant', dst_type=lambda x: x.decode())
-        if get_onnx_opset_version(node) < 11:
-            pads = onnx_attr(node, 'pads', 'ints', dst_type=lambda x: np.array(x, dtype=np.int64))
+        # Pytorch 1.3 while converting to opset 11, creates Pad from older opset.
+        # To be able to convert such models we have to check if pads attribute exists.
+        pads = onnx_attr(node, 'pads', 'ints', dst_type=int64_array)
+        if get_onnx_opset_version(node) < 11 or pads is not None:
             value = onnx_attr(node, 'value', 'f', default=0.)
```

Comment:
what will be if input type is integer then `value` also must be integer according to the specification?

Reply:
`value` is always float for Pad-2 according to [spec](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Pad-2).